client/app/achievements/achievements.html

```diff
@@ -11,34 +11,11 @@
 
 <div class="container">
   <div class="row text-area">
-    <div class="col-md-8">
```

Comment:
Can we just bump nav without the rest? It is probably not the most important thing to users just looking at our site for the first time.